helpers/shortcuts.ts

```diff
@@ -60,10 +60,16 @@ function transformKeys(keys: string[]) {
     if (Object.keys(keysMap).includes(key)) {
       return keysMap[key];
     }
-    return key.toUpperCase();
+    return toUpperCase(key);
   });
 }
 
+const toUpperCase = (key: string): string => {
```

Comment:
:bug: **Bug**
The `toUpperCase` function is incorrectly converting characters to uppercase. The current implementation subtracts 40 from the character code, which is incorrect. For example, the char code for 'a' is 97, and subtracting 40 gives 57, which corresponds to '9', not 'A'. The correct difference between the ASCII values of 'a' and 'A' is 32, not 40. Therefore, subtracting 32 would correctly convert a lowercase letter to uppercase. However, the best approach is to use the `String.prototype.toUpperCase()` method on the string to ensure accurate conversion.